README.md

```diff
@@ -1,6 +1,6 @@
 # Firefox 57 Campaign - heavily under construction (website and program)
 
-This project is based on https://mozilla.github.io/global-sprint/.
+This project is actually based on https://mozilla.github.io/global-sprint/.
```

Comment:
What exactly does this addition improve? I'm not a native speaker, but this seems like an unnecessary addition to me.

Rest looks good, thanks!